src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -100,6 +106,20 @@
     private static final int ONSTAGE_MIN_HEIGHT = 360;
 
     /**
+     * The ConfigurationService to get config values from.
+     */
+    private static final ConfigurationService
+        cfg = LibJitsi.getConfigurationService();
+
+    /**
+     * In order to limit the resolution changes due to bandwidth changes we
+     * react to bandwidth changes greater BWE_THRES / 100 of the last bandwidth
+     * estimation.
+     */
+    private static final int BWE_THRES
+        = cfg != null ? cfg.getInt(BWE_THRES_PNAME, 15) : 15;
```

Comment:
constant for default value?  also, we should have a unit of time here in the name i think (`BWE_CHANGETHRESHOLD_PCT`?) (i actually guessed _MS and _BPS first, but i think it's PCT? EDIT: and now actually i guess it isn't, since we still divide by 100....so i think units would help here :)